apps/mail/components/onboarding.tsx

```
@@ -101,34 +169,38 @@ export function OnboardingDialog({
-                      <Image
-                        priority
-                        width={500}
-                        height={500}
-                        src={step.video}
-                        alt={step.title}
-                        className="h-full w-full object-cover"
-                      />
-                    </div>
-                  ),
-              )}
+          {steps[currentStep] && steps[currentStep].video && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)